### PR TITLE
Support `pre-commit install` inside a worktree

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

**Comment by asottile** [comment]
Resolves #808

**Comment by asottile** [comment]
CC @s0undt3ch

**Comment by s0undt3ch** [comment]
Thanks!

**Comment by s0undt3ch** [comment]
Planning on releasing soon?

**Comment by asottile** [comment]
Yep, later today!

**Comment by asottile** [comment]
This has been released as part of 1.10.5 🎉

**Comment by s0undt3ch** [comment]
Awesome!

File: pre_commit/git.py [hunk 1]
```diff
@@ -31,10 +31,16 @@ def get_root():
...
-    ))
+    def _git_dir(opt):
+        return os.path.normpath(os.path.join(
+            git_root,
+            cmd_output('git', 'rev-parse', opt, cwd=git_root)[1].strip(),
+        ))
+
+    try:
+        return _git_dir('--git-common-dir')
+    except CalledProcessError:  # pragma: no cover (git < 2.5)
```
Comment (Heier2013) [inline pre_commit/git.py]: Consider compatibility git < 2.5?

**Pull request merged**